### ISSUE TITLE
Add email API for sending emails in production

### ISSUE THREAD

**Issue by harsh183** [dsc-umass/project-matching#24]
[Sendgrid](https://sendgrid.com/pricing/) has a free 100/emails a day pricing setup which looks fine, but there might be others as well that could be well suited. Heroku has good sendgrid integration too.

**Comment by AtrikDas** [comment]
Right now the email is being printed on the terminal but it's not being sent to the actual email client.

**Comment by harsh183** [comment]
Yup. Once this one has been added emails should be going out for real.
